test/unittests/main.cpp

```diff
@@ -0,0 +1,179 @@
+// evmone: Fast Ethereum Virtual Machine implementation
```

Comment:
It would probably be better if this file were in a separate directory, not `unittests`?

Reply:
There are more changes to dir structure needed, so I left it for another time.

Preferably the `evm-test` should be in `tools/`, not in `test/` but because the `evm-unitests` is shared between `evm-test` and `evmone-unittests` it might be overkill.

Maybe just get rid of `test/` dir entirely...